neuralforecast/common/_modules.py

```diff
@@ -486,7 +487,87 @@ def forward(self, x, x_mark=None):
 
         return self.dropout(x)
 
-# %% ../../nbs/common.modules.ipynb 19
+# %% ../../nbs/common.modules.ipynb 20
+class Normalize(nn.Module):
```

Comment:
Can we name this class RevIN (as that's what it does)? Note that normalization isn't preferred in this location; ideally we have the normalization steps in common.scalers.ipynb (which also has revin), however for the case of RevIN specifically this has been somewhat hard to implement.

Reply:
I see that I was a bit ambiguous here. In `_scalers`, we include scalers that scale the data prior to entering the NN, such as `standard` and `robust`. One could see `RevIN` as such a scaler (which I think it ultimately is). The comment above implied that if we collect all `RevIN` implementations, we could remove them from their original models and just have a user choose a scaler upon training the model, for which one of the options is `RevIN`. 

However, the current PR justs collects the existing RevIN modules into a common file, for which `_modules.py` is more appropriate. Makes sense what I mean? At the time of the comment I thought the idea of this PR was to fully integrate `RevIN` as a scaler (not as a module).